libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -267,6 +267,7 @@ string IRGenerator::dispatchRoutine(ContractDefinition const& _contract)
 			</cases>
 			default {}
 		}
+		if iszero(calldatasize()) { <receiveEther> }
```

Comment:
The contrast between the two implementations is almost ridiculous

Reply:
Yea.. wish we could use this one already